actions/invite_actions.jsx

```diff
@@ -59,7 +59,7 @@ export function sendMembersInvites(teamId, users, emails) {
                 }
             } else {
                 for (const email of emails) {
-                    const inviteWithError = invitesWithErrors.find((i) => email === i.email && i.error);
+                    const inviteWithError = invitesWithErrors.find((i) => email.toLowerCase() === i.email && i.error);
```

Comment:
Test kept failing here returning a success message when it should return a failure. It was because if you entered something like `IAmSoFreakingGood@mattermost.com` , server returns `iamsofreakinggood@mattermost.com` (all lowercase) and so the matching failed.